js/cards.js

```diff
@@ -66,13 +71,19 @@ export async function defineCardComponent() {
         (card) => card["Collector Number"] == this.getAttribute("card-id")
       );
       this.innerHTML = html;
+      this.holder = this.getElementsByClassName("card-component")[0];
       this.front = this.getElementsByClassName("card-front")[0];
       this.back = this.getElementsByClassName("card-back")[0];
-      const image = this.getElementsByClassName("card-image")[0];
-      image.src = this.getImageURL();
+      this.info = this.getElementsByClassName("card-info")[0];
+      this.image = this.getElementsByClassName("card-image")[0];
+      // const image = this.getElementsByClassName("card-image")[0];
```

Comment:
delete commented code

Reply:
Missed this while scrolling, thank you for the pickup!